src/gui/transferlistdelegate.cpp

```diff
@@ -38,183 +37,32 @@
 #include <QProxyStyle>
 #endif
 
-#include "base/bittorrent/torrenthandle.h"
-#include "base/preferences.h"
-#include "base/types.h"
-#include "base/unicodestrings.h"
-#include "base/utils/misc.h"
 #include "base/utils/string.h"
 #include "transferlistmodel.h"
 
 TransferListDelegate::TransferListDelegate(QObject *parent)
-    : QItemDelegate(parent)
+    : QStyledItemDelegate {parent}
 {
 }
 
 void TransferListDelegate::paint(QPainter *painter, const QStyleOptionViewItem &option, const QModelIndex &index) const
 {
+    if (index.column() != TransferListModel::TR_PROGRESS)
+        return QStyledItemDelegate::paint(painter, option, index);
```

Comment:
Do you have a prejudice against such a notation? It's valid. 
I agree, it does not make sense in the general case. But when we need to call void function immediately followed by return statement, I find it very convenient to write `return voidFunction();`. In addition, it makes all similar "delegation to another function" cases look uniform.

Reply:
>it does not make sense in the general case. 

For me it doesn't make sense in every cases.

Reply:
Just different subjective opinions...
So if you want to never use some kind of statement we need discuss it to forbid in our Coding Guidelines.

Reply:
I wouldn't try to push to the extreme just to satisfy my taste if I were you but let us also hear what others think.

Reply:
IMO, since we can do:
```c++
bool doSomething()
{
  return didSomethingElse();
}
```

I can't find a particular reason to object to the same thing being done for function returning void. Although I do agree that it looks rather weird.